src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -141,7 +141,8 @@ private void createTemplateFeedbackQuestions(String courseId, String feedbackSes
         return new ArrayList<FeedbackQuestionAttributes>();
     }
 
-    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+    @Override
+    protected FeedbackSessionAttributes extractFeedbackSessionDataSetUniqueAttributes(
```

Comment:
Is the `extractFeedbackSessionData` part of the method name necessary? It seems like you're simply setting attributes.